### ISSUE TITLE
Pomatez size window

### ISSUE THREAD

**Issue by madkoding** [roldanjr/pomatez#132]
**Is your feature request related to a problem? Please describe.**
Is way too big for small screens like 1280x720

**Describe the solution you'd like**
Add an option to reduce the size almost a half

**Describe alternatives you've considered**
As an alternative, a third layout semitransparent

**Comment by roldanjr** [comment]
@madkoding yeah soon. I will try to make it friendly to all desktop screen sizes.

But for now, in my case I have no enough time to do that.

I will add it on app's roadmap.

**Comment by dembeck** [comment]
I also would really like it if the window was resizable, but I want to make it bigger so that I can see more of the task details (where I put small sub-tasks) on the timer screen.

An app that could provide some inspiration for how layout might work as the window resizes is Plexamp. It is normally a smaller window, but as you make the window bigger, it shows more information, but is still designed to be a small window.

![image](https://user-images.githubusercontent.com/9568447/109216580-8e716680-7769-11eb-9a3d-1067a7219328.png)
![image](https://user-images.githubusercontent.com/9568447/109216610-992bfb80-7769-11eb-831a-829cba9ac11a.png)
![image](https://user-images.githubusercontent.com/9568447/109217280-8108ac00-776a-11eb-928c-0ef702f951c0.png)

**Comment by SirNate0** [comment]
I'd also like to second the request to be able to resize the window. For me, it is particularly the width - I would like to be able to make the window wider to show the full title of the current card from the task list. Or the title should wrap as it is displayed in the task list, and then just making the window taller might work.

**Comment by jamin-peckham-sp** [comment]
Also +1. I use the task list and would like to be able to see more of the task list by resizing the window vertically.

**Comment by sekwah41** [comment]
Tbh I kind of want to redo the CSS for the layout to make it resizable anyway. I can try to take a look at it when I get some time.
